cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java

```diff
@@ -9,9 +9,9 @@
  */
 public class DebugUtils {
 
-    private static final String DEBUG_ARGS_JAVA8 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
```

Comment:
Seems like debugging will only support in Java 11. Don't we have to support both Java 8 and Java 11?